<launch>
    <node pkg="sunwenbin_practices" name="robot_follow" type="robot_following.py" output="screen">
        <rosparam>
            haar_scaleFactor: 1.2
            haar_minNeighbors: 2
            haar_minSize: 40
            haar_maxSize: 60
        </rosparam>
        <param name="cascade_1" value="$(find sunwenbin_practices)/data/haar_detectors/haarcascade_frontalface_alt.xml" />
        <param name="cascade_2" value="$(find sunwenbin_practices)/data/haar_detectors/haarcascade_profileface.xml" />
    </node>
    <node name="face_compressed_image" pkg="image_transport" type="republish" args="raw in:=face_image compressed out:=face_image" />
</launch>
